apps/scratch/src/pages/__tests__/page-objects/scratch-editor.ts

```diff
@@ -98,6 +99,13 @@ export class ScratchEditorPage {
     return this.openTaskConfigButton.click();
   }
 
+  async pressGreenFlag() {
+    return this.page
+      .getByTestId("stage-controls")
+      .locator("img:first-child")
```

Comment:
no, unfortunately not. we would need to copy over the "Controls" component to add testids. we could do that but I think I would rather go with this for now to avoid having too much scratch code copied over?